docs/tx/processing.md

```diff
@@ -27,14 +27,14 @@
 
 ## Prerequisites
 
-* You should know what a transaction is, its structure and about tx
-  witnesses. You can read about it
+* You should know what a transaction is, be familiar with its structure, and know about tx
+  witnesses and payloads. You can read about these concepts here:
   [here](https://cardanodocs.com/cardano/transactions/).
-* You also should know technical details about
-  [addresses](https://cardanodocs.com/cardano/addresses/).
-* Please read about
+* You should also know technical details about
+  [addresses], see: (https://cardanodocs.com/cardano/addresses/).
```

Comment:
I don't think this is correct markdown syntax. Note that `[something](somelink)`is actually markdown syntax.